universal-application-tool-0.0.1/app/views/admin/ProgramEditView.java

```diff
@@ -0,0 +1,31 @@
+package views.admin;
+
+import static j2html.TagCreator.body;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.join;
+
+import play.mvc.Http.Request;
+import play.twirl.api.Content;
+import services.program.ProgramDefinition;
+import views.BaseHtmlView;
+
+public class ProgramEditView extends BaseHtmlView {
```

Comment:
suggestion (blocking): we'll need to inject `BaseHtmlLayout` here and use `layout.htmlContent()` below

Reply:
Yep - you'll get a compiler error when you pull the latest changes from main.

Eventually, this should inject something like `AdminLayout` (which will extend `BaseHtmlLayout`.

Reply:
added a placeholder `AdminProgramLayout` and added it to the admin program views.